packages/@okta/i18n/src/properties/login.properties

```diff
@@ -312,6 +312,7 @@ email.button.send = Send me the code
 email.button.resend = Send again
 email.code.label = Verification code
 email.code.not.received = Haven\'t received an email?
+idx.email.code.not.received = Haven\'t received an email?
```

Comment:
Why do we need a second key? If possible, let's try to reuse `email.code.not.received` (and our other properties) and [map them accordingly](https://github.com/okta/okta-signin-widget/blob/5ae7f2bec639af5a7c58b1c0db8d82c8e358489d/src/v2/ion/i18nTransformer.js#L56).

Reply:
i think because backend is sending a new key

Reply:
The backend can always send a different key, the Widget just needs to map this to a v1 key in our `i18nTransformer`. Best practice is to reuse existing translations whenever the flows are the same, especially when the values are identical.

Reply:
Agree. backend is not yet merged, we can ask them to use the existing key i.e `email.code.not.received`

Reply:
Looking at the backend PR here: https://github.com/okta/okta-core/pull/55383/files#diff-4ece32bc1f9d5ffcaf775429ce7813a08e7ec799bc1f74f2875ad43a0962ff2cR1037 - I think they use an existing pattern of starting their keys with the "idx" prefix so it might not make sense to ask them to change it just for the UI. 

In our case here, having the i18n mapping won't be enough either because in https://github.com/okta/okta-signin-widget/pull/2158/files#diff-ef082e5c8e18a4aa6fe3c57416549703855d9bbea222b747cb5e846b541185c6R17 we check the message key against the response we get from the server and the key in there won't use the mapping value.

Reply:
We have a convention for this already, using the [`getI18nKey` utility](https://github.com/okta/okta-signin-widget/blob/56930aa682bf8e0aee294b5a1c8588c5b0d4c6a8/src/v2/ion/i18nTransformer.js#L249-L269) to pick-up the mapped value.

Our goal is to reduce the number of breaking changes for customers migrating from classic to OIE. Specifying the map to a classic key allows them to continue using their customizations.

Reply:
Yeah that's a good point. I've updated the solution now so that it does re-use the v1 key. Please have a look again